src/Compilers/CSharp/Portable/Symbols/ObsoleteAttributeHelpers.cs

```diff
@@ -154,12 +155,19 @@ static DiagnosticInfo createObsoleteDiagnostic(Symbol symbol, BinderFlags locati
                     return null;
                 }
 
-                if (data.Kind == ObsoleteAttributeKind.Experimental)
+                if (data.Kind == ObsoleteAttributeKind.WindowsExperimental)
                 {
                     Debug.Assert(data.Message == null);
                     Debug.Assert(!data.IsError);
-                    // Provide an explicit format for fully-qualified type names.
```

Comment:
>Provide an explicit format for fully-qualified type names. [](http://example.com/codeflow?start=23&length=58)

Are we testing that type names are fully-qualified? (The format used here is `CSharpShortErrorMessageFormat` which may drop the namespace.)